index.html

```diff
@@ -5041,16 +5041,19 @@ <h5>Note regarding the ARIA 1.1 <rref>none</rref> role.</h5>
 				<section id="conflict_resolution_presentation_none">
 					<h5>Presentational Roles Conflict Resolution</h5>
 					<p>There are a number of ways presentational role conflicts are resolved.</p>
-					<p>Host languages elements having implicit presentational roles for which no roles may be applied MUST never be exposed in the accessibility tree, with this exception: user agents MUST always expose global WAI-ARIA states and properties to accessibility APIs. In this case, the user agent ignores the <code>presentation</code> role and exposes the element according to its implicit native semantics. However, user agents MUST ignore any non-global, role-specific WAI-ARIA states and properties, unless it is on an inherited presentational role where an explicit role is applied.</p>
+					<p>Host language elements having explicit or inherited presentational role MUST NOT be exposed in the accessibility tree, with these exceptions:</p>
+					<ul>
+						<li>If an element is focusable, or otherwise interactive, user agents MUST ignore the <code>presentation</code> role and expose the element with its implicit native semantics, in order to ensure that the element is both <a>understandable</a> and <a>operable</a>.</li>
+						<li>If an element has global WAI-ARIA states or properties, user agents MUST ignore the <code>presentation</code> role and expose the element with its implicit native semantics. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
```

Comment:
Why do we need "unless the presentational role is inherited and an explicit non-presentational role is applied." 
If there is an explicit non-presentational role then this whole section would not apply as the section is about presentational roles conflict resolution.

Reply:
I think it was intended to clarify that an explicit role wins over an inherited presentational role.
So if an author wanted to use `<table role="presentation">` for layout, with `<td role="checkbox">` descendants, then they would know that the td's inherited role of presentation is ignored and "checkbox" applies.

However, you are correct that it has nothing to do with any non-global attributes (i.e. `aria-checked` for example), so the "clarifying" phrase is in the wrong place.

Perhaps it merits a separate bullet? i.e. remove the phrase from where it currently is, and instead add:
- If a required owned element has an explicit non-presentational role, user agents MUST ignore an inherited presentational role

If you agree, then I would make it the 2nd of the 3 points (so that the point about global/non-global attributes comes last, right before Example 12).

Reply:
Got it. I think it would actually be clearer to completely delete that paragraph, remove the phrase, and insert the following bullet in the "MUST NOT be exposed in the accessibility tree, with these exceptions" list (because I think it is another exception, and should be in the list):

- If a required owned element has an explicit non-presentational role, user agents MUST ignore an inherited presentational role and expose the element with its explicit role. If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined and the user agent MAY resort to an internal recovery mechanism to repair the accessibility tree.

Here's a preview of the [Presentational Roles Conflict Resolution section](https://pr-preview.s3.amazonaws.com/w3c/aria/pull/852.html#conflict_resolution_presentation_none) with those changes.